### DISCUSSION TITLE
Git Bash can affect from Apache log4j2 vulnerability or not

### DISCUSSION THREAD

**Discussion by Shweta-Garbhe** [community/community#22362] [New to GitHub]
<p>I am using GIT bash since long time.<br>
Recently, we all are facing Apache log4j2 vulnerability related issues. In context of the same, I wanted to know is GIT bash is affected by any log4j related files.<br>
Could any one confirm that GIT is vulnerable from Apache log4j2 or not.</p>

**Comment by airtower-luna** [comment]
<p>Git itself is written in C, so it’s definitely not using log4j at all. :slightly_smiling_face:</p>
<p>Other software built around Git might be affected, if it has Java components. If you use anything like that, you’ll have to check news specifically for that software. For example GitHub has released some updates for their enterprise server, see here: <a href="https://github.blog/2021-12-13-githubs-response-to-log4j-vulnerability-cve-2021-44228/" class="inline-onebox">GitHub’s response to Log4j vulnerability CVE-2021-44228 | The GitHub Blog</a></p>

**Comment by Shweta-Garbhe** [comment]
<p>Thanks <a class="mention" href="/airtower-luna">@airtower-luna</a> for your answer but I want to know is your answer is applicable to Git Bash ( versioning tool - <a href="https://git-scm.com/downloads" class="inline-onebox" rel="noopener nofollow ugc">Git - Downloads</a> ) as well.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Git Bash is just a variant of the Bash shell that’s packaged with Git for Windows so you have a Unix/Linux-like shell to work with. No Java there either.</p>
